src/App/App.stories.tsx

```diff
@@ -1,6 +1,6 @@
 import type { Meta, Story } from '@storybook/react';
 
-import { GetAdminsIdQuery } from '../operations/queries';
+import { GetAdminListDocument } from '../operations/__generated__/typesAndHooks';
```

Comment:
If we re-export it in 'operations,' modifying the structure will be less expensive.

Reply:
I thought that it's a good idea to see __generated__ in the importing path
to know that these are generated files
but really we don't need to know about it so I'll re-export